extensions/api/identity-api/api-configuration/src/main/resources/identity-api-version.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "version": "1.0.0-alpha",
+    "urlPath": "/v1alpha",
+    "lastUpdated": "2024-08-22T09:20:00Z",
+    "maturity": "alpha"
```

Comment:
I think the api verification it's breaking because, to understand which item of the version file to use, it's looking for either "maturity is stable or maturity is null". So, in the case of a single version, it should be stable or null.
Not the best, but putting in place something that decide which maturity is the best maybe would be a little overkill

Reply:
ok, then `maturity = null` is the way to go